WPMedia/fusion-news-theme-blocks
# Themes 1184

## Description

This PR updates the logical properties on the promo blocks (non-manual). The small promo block does not need any changes.

## Jira Ticket

- [THEMES-1184](https://arcpublishing.atlassian.net/browse/THEMES-1184)

## Acceptance Criteria

1. The above logical properties are applied and tested on the promo blocks.
2. Storybook is updated with RTL. 

## Test Steps

1. Checkout this branch `git checkout THEMES-1184`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/small-promo-block,@wpmedia/medium-promo-block,@wpmedia/large-promo-block,@wpmedia/extra-large-promo-block`
4. Inspect the small, medium, large, and xl promo blocks and see that they are using logical properties in the correct places.
5. It should match the designs in Figma.
6. Set `"textDirection": "rtl"` in the `"siteProperties"` object of the `blocks.json`.
7. Restart PageBuilder and verify that the styling adapts to the new text direction.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1184]: https://arcpublishing.atlassian.net/browse/THEMES-1184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
